src/components/Calculator.js

```diff
@@ -0,0 +1,49 @@
+import React from 'react';
+
+// eslint-disable-next-line react/prefer-stateless-function
```

Comment:
## OPTIONAL


- [ ] Kindly avoid disabling linters as it is not a good practice, it is better for your linters to pass the linter checks 👍🏿